src/engine/source/indexerconnector/include/indexerConnector.hpp

```diff
@@ -22,9 +22,10 @@
 #define EXPORTED
 #endif
 
-static constexpr auto DEFAULT_INTERVAL = 60u;
+static constexpr auto DEFAULT_INTERVAL = 60000u;
```

Comment:
Question: Why did you change the interval?

Reply:
Time scale. We changed the duration from seconds to milliseconds, in order to get the same time window, we had to adapt this value.
The reason: The test needs to pass in parallel, and with a high window interval (15s per test) this is not possible. Now, the duration of all tests has been reduced to 17ms.